include/deal.II/physics/transformations.h

```diff
@@ -29,6 +30,73 @@ namespace Physics
   {
 
     /**
+     * Transformation functions and tensors that are defined in terms of
+     * rotation angles and axes of rotation.
+     *
+     * @author Jean-Paul Pelteret, 2017
+    */
+    namespace Angular
```

Comment:
That's an adjective, but we've usually named namespaces as nouns. Since you're dealing with rotations etc, why not just call the namespace `Rotations` or similar?